src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_after_failing_delegatecallFiller.json

```diff
@@ -8,12 +8,6 @@
             "currentTimestamp" : "0x03e8"
         },
         "pre" : {
-            "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
-                "balance": "0x100000",
-                "code": "",
```

Comment:
so I think here would have been the call to 100...1 which fails. and then sstore[1] = 1 to indicate that the test worked